pkg/gardenlet/operation/botanist/nodelocaldns.go

```diff
@@ -53,9 +55,10 @@ func (b *Botanist) ReconcileNodeLocalDNS(ctx context.Context) error {
 	} else {
 		dnsServers = coreDNS
 	}
+	ipfamilies = b.Shoot.GetInfo().Spec.Networking.IPFamilies
 	b.Shoot.Components.SystemComponents.NodeLocalDNS.SetClusterDNS(clusterDNS)
 	b.Shoot.Components.SystemComponents.NodeLocalDNS.SetDNSServers(dnsServers)
-
+	b.Shoot.Components.SystemComponents.NodeLocalDNS.SetIPFamilies(ipfamilies)
```

Comment:
Please inline directly. There is no use in having the variable.

```suggestion
	b.Shoot.Components.SystemComponents.NodeLocalDNS.SetIPFamilies(b.Shoot.GetInfo().Spec.Networking.IPFamilies)
```